### ISSUE TITLE
covoiturage-libre en Suisse?

### ISSUE THREAD

**Issue by paulpell** [covoiturage-libre/covoiturage-libre#167]
Bonjour!

## Introduction
Petite présentation: je m'appelle Paul et habite en Suisse (vers la Franche-Comté et le Doubs). Bon ..
J'écris en français: il ne s'agit pas d'une discussion technique..

Nous sommes deux personnes à vouloir mettre en place une plateforme de covoiturage dans notre région, qui n'est pas très bien desservie en transports publics.
Je m'occupe de l'aspect technique et l'autre personne plutôt de communication et toutes ces petites choses à côté (rôles non stricts :) ). Nous en sommes encore aux débuts (phase de recherche, réflexion, etc.). Nous pensons aujourd'hui à faire cela sous forme d'une association à but non lucratif.

J'ai trouvé votre projet intéressant:
* techniquement et source libre
* politiquement / philosophiquement

Je souhaiterais donc réutiliser votre code.. Nous aimerions cibler d'une part les trajets liés à des événements précis (ex. festivals locaux) et d'autre part les trajets quotidiens (travail, ..).

Je pense que pour avoir un impact par chez nous, il vaut mieux utiliser un site (dans le sens front-end) "personnalisé régionalement" et non pas utiliser directement covoiturage-libre.fr

## Les questions
Dans l'idée, êtes-vous d'accord que nous nous basions sur votre code?
À mon sens, je me joindrais à vous pour développer covoiturage-libre.

Que pensez-vous si nous avons un propre serveur qui tourne avec covoiturage-libre?

Ce serait super!! 
Discussion à suivre?

**Comment by nicolasleger** [comment]
Bonjour,

Votre besoin rejoint ce que j'ai initié récemment pour déployer plus facilement et générer d'autres plateformes https://github.com/covoiturage-libre/covoiturage-libre/pull/160

À discuter avec l'équipe

Bien à vous

**Comment by Poussinou** [comment]
Bonjour, @paulpell
Tout d'abord merci de prendre contact avec nous, c'est toujours mieux de discuter :-)
L'association n'est pas contre l'export de notre code à l'étranger pour créer d'autres plateformes, c'est toujours mieux d'être présent à l'international, encore plus si on considère que la Suisse est frontalière de la France.
Cependant, nous voulons que toutes les annonces soient centralisées, cad que si je fais une recherche depuis covoiturage-libre.fr pour me déplacer dans votre région suisse, je puisse trouver un trajet (qui aurait été déposé via votre site). Il faut encore qu'on travaille sur ça, peut être cela pourrait être fait via une API. Nous voulons de plus limiter au maximum la multiplication de plateformes basées sur CVL, même si elles vont toutes piocher dans la même base de donnée, donc éventuellement si votre site pourra dans le futur viser plus ou moins toute la suisse, ça serait cool mais on en est pas encore là.

Pour récapituler, nous sommes plutôt pour votre projet MAIS seulement quand on aura ce moyen de partager les annonces entre les différentes instances. Et aussi évidemment le fait que votre plateforme ne vise pas le profit, pas de frais obligatoires pour qui que ce soit. On en reparlera... ;-)

Et comme le précise @nicolasleger il y a en effet une discussion pour faciliter le développement et le déploiement du site

**Comment by paulpell** [comment]
Merci @Poussinou @nicolasleger 

Nous nous retrouvons (avec cette autre personne) en fin de semaine, nous vous en dirons plus.
Cela a tout son sens selon moi d'avoir tous les trajets soit centralisés, soit à disposition des différentes instances.

**Comment by paulpell** [comment]
Bonjour à tous!

Nous en avons parlé un peu, et nous ne sommes pas encore fixés sur beaucoup d'aspects. Travail en progression!

Ma proposition aujourd'hui est que nous utilisions CVL dans le cadre d'une ou deux manifestations locales, en partenariat avec eux. Cela signifie qu'on peut déployer l'application pour quelques semaines seulement, histoire pour moi de me faire la main. Cela nous permettrait aussi d'évaluer la réaction des gens, l'attractivité, etc.. Enfin, de se faire une idée.
**SI** ça prend et que des gens s'inscrivent, nous aurions alors également un serveur avec quelques données pour tester/développer des échanges entre base de données.

Comment voyez-vous cela?

**Comment by nicolasleger** [comment]
Bonjour,

Docker / Heroku, voir PRs en cours mais il faut penser à un système de template et renommage de l'application.

**Comment by Poussinou** [comment]
Salut @paulpell je ne suis pas contre le déploiement local pour une durée limitée pour quelques événements (avec comme objectif de faire un site national plus tard ;p )
Nous écouterons vos retours sur est-ce que ça a marché ou pas avec intérêt !
Cependant il n'y a pas de système d'inscription en ce moment, le dépôt et la consultation d'annonces est libre, sans compte, vous ne pourrez avoir des retours qu'en demandant directement aux gens...
OK pour travailler ensemble ensuite sur les interactions entre les différentes plateforme, avec plaisir :)

**Comment by Poussinou** [comment]
@paulpell Qu'en a t'il été finalement? 😃

**Comment by paulpell** [comment]
@Poussinou Nous avons mis en pause pour différentes raisons. Nous avons rencontré plusieurs acteurs (festivals, autorités) et pour résumer tout le monde répond:
"Ah ouais ce serait super de faire ça, débrouillez-vous" ... En plusieurs phrases, il faut leur laisser :)
Et ma collègue commence des cours du soir dans un autre domaine.

Merci de me relancer! Ces derniers temps j'y repense et je me dis que c'est bête de laisser tomber.
Je ne suis pas encore mûr, je crois.. mais il faut quand même se dépécher avant que le mot covoiturage soit remplacé par un nom de marque ou d'entreprise, vous avez raison là-dessus!

De votre côté, comment est-ce que ça va? Le non-technique, j'entends :)

**Comment by Poussinou** [comment]
@paulpell Oui, je vois le problème... Si ils ne soutiennent pas un minimum la démarche c'est sûr que ça sera compliqué...
C'est bête de laisser tomber. Je pense que dans l'idée, comme on disait plus haut, on est d'accord pour que les festivals s'appuient sur notre site !
Il n'y a pas de besoin de précipiter les choses, je considère à titre tout à fait personnel que le temps joue pour nous ;) Vous n'avez pas Blablacar en Suisse?

Nous ça va écoute, on a @webplate @nicolasleger qui ont vachement avancé cet été, notamment sur la facilité à dépolyer et travailler sur le code source du site. Au delà rien de nouveaux :)

**Comment by AlexBillbao** [comment]
Ça serait clairement super de monter des partenariats transfrontaliers, et en premier lieu avec la suisse (et la Belgique), car pour la partie Suisse-Est il n'y a pas la barrière de la langue  !

Peut-être commencer par acheter le nom de domaine [covoiturage-libre.ch](https://covoiturage-libre.ch/) serait un début ?

Comme ça on pourrait communiquer en Suisse avec ce nom de domaine, qui rabattrait simplement sur le site.fr et donc sur sa base de données ...

Concernant la duplication de la plateforme (2 bases de données chez 2 hébergeurs), c'est clairement scier la branche sur laquelle on est assis tant qu'on a pas une API de partage entre base de données. Et quand bien même on l'aurait, est-ce vraiment intéressant ?
Pour payer 2 serveurs pour un projet commun ?

Débat à poursuivre !
